README.md

```diff
@@ -8,10 +8,31 @@ The stack also creates a small EC2 instance (defined in ec2-test.tf) that will b
 
 ## Deployment
 
+This stack is meant to be consumed as a module in your existing terraform stack. You can consume it by using code similar to this:
+
+    ````
+    module "ekk_stack" {
+        source = "github.com/GSA/devsecops-ekk-stack"
+        s3_logging_bucket_name = "${var.s3_logging_bucket_name}"
+        es_kinesis_delivery_stream = "${var.es_kinesis_delivery_stream}"
+    }
+    ````
```

Comment:
Minor: de-indent this block, use three backticks, and `hcl` after the top three to get syntax highlighting.

Reply:
Fixed.